packages/vscode-extension/src/debugging/startDebugging.ts

```diff
@@ -0,0 +1,41 @@
+import assert from "assert";
+import * as vscode from "vscode";
+import { debug, Disposable } from "vscode";
+
+/**
+ * Helper function that starts a debug session and returns the session object upon sucesfull start
+ */
+export async function startDebugging(
+  folder: vscode.WorkspaceFolder | undefined,
+  nameOrConfiguration: string | vscode.DebugConfiguration,
+  parentSessionOrOptions?: vscode.DebugSession | vscode.DebugSessionOptions
+) {
```

Comment:
NIT:
```suggestion
): Promise<vscode.DebugSession> {
```
I know TS can infer this, but when I was reading this PR it wasn't clear to me:
1. if this function returns something, or is just used for side-effects, due to the `return` statement being buried three indentations deep.
2. if it always returns the session, or can return `undefined` in some cases.